src/components/shared/StatsList.tsx

```diff
@@ -0,0 +1,24 @@
+import { cn } from '@/lib/utils'
+import React from 'react'
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  separator?: boolean
+}
+
+export const StatsList: React.FC<Props> = ({ separator, className, children, ...props }) => {
+  const items = React.Children.toArray(children)
+
+  return (
+    <div
```

Comment:
can we just render children here?
and get rid of `separator` property - feels like we don't need it

Reply:
I've added separator since the designs differ if there are 3 or 4 rows of data, I propose we keep this? It will come in handy as API endpoints change.
The component doesn't throw errors/malfunction if there is only one child btw.

Reply:
I think we can send separator inside the `children` props.
```
    <StatsList>
      <StatsItem  />
      <Separator />
      <StatsItem  />
      <StatsItem  />
    </StatsList>
    ```

Reply:
Does it make sense then to keep the stats list as a separate component? maybe just put a CSS name tag at the end of div? Like "STATSITEMWRAPPER"?

Reply:
Or put a separator component inside StatsList file?

Reply:
IMO keep it as `StatsList`, `StatsContainer`, maybe `StatsContainer` sounds better since it's just container and styling around

Reply:
> Or put a separator component inside StatsList file?

if separator will come after each item you can either send prop to StatsItem like `withSeparator` or just add it manually. We have `Divider` component already I believe